features/registering_assets.feature

```diff
@@ -12,8 +12,8 @@ Feature: Registering Assets
 
   Scenario: Viewing default asset files
     When I am on the index page for posts
-    Then I should see the css file "active_admin.css"
-    Then I should see the js file "active_admin.js"
+    Then I should see the css file "active_admin"
```

Comment:
Because I removed asset.digest = false, so the css file is now something like active_admin.87a9...98ef.css